xtensa-lx-rt/src/exception/context.rs

```diff
@@ -134,6 +134,14 @@ unsafe extern "C" fn __default_exception(cause: ExceptionCause, save_frame: &mut
 #[no_mangle]
 #[link_section = ".rwtext"]
 extern "C" fn __default_user_exception(cause: ExceptionCause, save_frame: &Context) {
+    #[cfg(any(feature = "esp32", feature = "esp32s3"))]
+    if cause == ExceptionCause::Cp0Disabled {
```

Comment:
I don't have a preference but you can do this to skip the `PartialEq`.
```suggestion
    if matches!(cause, ExceptionCause::Cp0Disabled) {
```